youtube_dl/extractor/jamendo.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+
+
+class JamendoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?jamendo\.com/track/(?P<id>[0-9]+)/(?P<display_id>[\w-]+)'
+    _TEST = {
+        'url': 'https://www.jamendo.com/track/196219/stories-from-emona-i',
+        'md5': '697564d8b10c60af35edda4fafb3fbfd',
+        'info_dict': {
+            'id': '196219',
+            'display_id': 'stories-from-emona-i',
+            'ext': 'mp3',
+            'title': 'Stories from Emona I',
+            'thumbnail': 're:^https?://.*\.jpg',
+            'url': 'https://mp3d.jamendo.com/download/track/196219/mp32'
+        }
+    }
+
+    def _real_extract(self, url):
+        url_data = self._VALID_URL_RE.match(url)
+        track_id = url_data.group('id')
+        webpage = self._download_webpage(url, track_id)
+
+        thumbnail = self._html_search_regex(r'<meta itemprop="image" content="(.+)">', webpage, 'thumbnail', fatal=False)
+        title = self._html_search_regex('<meta itemprop="name" content="(.+)">', webpage, 'title')
+
+        return {
+            'id': track_id,
+            'display_id': url_data.group('display_id'),
+            'ext': 'mp3',
+            'title': title,
+            'thumbnail': thumbnail,
+            'url': 'https://mp3d.jamendo.com/download/track/%s/mp32' % track_id
```

Comment:
Hm, the official API is redirecting to `https://mp3d.jamendo.com/download/track/677050/mp32/` for `https://api.jamendo.com/v3.0/tracks/file/?client_id=56d30c95&audioformat=mp32&id=677050` (and similar ones for `flac`, `ogg` and `mp31`)